bukkit/src/main/java/net/william278/huskhomes/BukkitHuskHomes.java

```diff
@@ -174,9 +174,12 @@ public void onEnable() {
 
             // Register plugin hooks (Economy, Maps, Plan)
             this.pluginHooks = new HashSet<>();
-            if (settings.economy && Bukkit.getPluginManager().getPlugin("Vault") != null) {
-                pluginHooks.add(new VaultEconomyHook(this));
-            }
+            if (settings.economy)
```

Comment:
Missing braces here, should be `if (settings.economy) { ... }`

Reply:
k